tutorials-and-examples/example-notebooks/Example - Using Sentinel Search Queries.ipynb

```diff
@@ -0,0 +1,361 @@
+{
```

Comment:
<p>Won't this run forever? Maybe have a timeout with a break</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Azure/Azure-Sentinel-Notebooks/pull/144/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>